src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.IntegrationTests/CustomWebApplicationFactory.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+
+namespace Altinn.Platform.Authentication.IntegrationTests
+{
+    /// <inheritdoc />
+    public class CustomWebApplicationFactory<TStartup> : WebApplicationFactory<TStartup>
```

Comment:
I now think the CustomWebApplicationFactory class is unnecessary. Can probably use the WebApplicationFactory directly.